crates/pallet-subspace/src/lib.rs

```diff
@@ -1116,10 +1116,6 @@ impl<T: Config> Pallet<T> {
     fn do_enable_rewards_at(
         enable_rewards_at: EnableRewardsAt<BlockNumberFor<T>>,
     ) -> DispatchResult {
-        if EnableRewards::<T>::get().is_some() {
-            return Err(Error::<T>::RewardsAlreadyEnabled.into());
```

Comment:
If you remove the case that triggers it, then also remove corresponding enum variant from `Error`, though I don't know if we need to maintain variant indices in SCALE encoding, maybe @vedhavyas knows

Reply:
Its fine to remove the error variant. 
Shouldn't be any scale issue within the runtime.
I dont see any usages of this variant outside runtime

Reply:
IMO we can remove the whole ability to set/reset rewards and solution range adjustment shortly.